apps/browser/browserservice.cpp

```diff
@@ -201,40 +198,18 @@ void BrowserUIService::requestTab(int tabId, const QString &url)
 void BrowserUIService::closeTab(int tabId)
 {
     // Let privileged and the process that created the tab to also destroy it
```

Comment:
I think you should adjust this comment too.

Reply:
Thanks for noticing, I've changed it.